tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -467,22 +468,44 @@ public function adminHasAddedUserToGroupUsingTheGraphApi(
 	}
 
 	/**
-	 * @When /^the administrator creates a group "([^"]*)" using the Graph API$/
 	 *
 	 * @param string $group
+	 * @param ?string $user
 	 *
-	 * @return void
-	 * @throws Exception
+	 * @return ResponseInterface
 	 * @throws GuzzleException
 	 */
-	public function adminCreatesGroupUsingTheGraphApi(string $group): void {
-		$response = GraphHelper::createGroup(
+	public function createGroup(string $group, ?string $user = null): ResponseInterface {
+		if ($user) {
+			$username = $user;
+			$password = $this->featureContext->getPasswordForUser($user);
+		} else {
+			$username = $this->featureContext->getAdminUsername();
+			$password = $this->featureContext->getAdminPassword();
+		}
+		return GraphHelper::createGroup(
 			$this->featureContext->getBaseUrl(),
 			$this->featureContext->getStepLineRef(),
-			$this->featureContext->getAdminUsername(),
-			$this->featureContext->getAdminPassword(),
+			$username,
+			$password,
 			$group,
 		);
+	}
+
+	/**
+	 * @When /^the administrator creates a group "([^"]*)" using the Graph API$/
+	 * @When user :user creates a group :group using the Graph API
+	 * @When user :user tries to create a group :group using the Graph API
+	 *
+	 * @param string $group
+	 * @param ?string $user
+	 *
+	 * @return void
+	 * @throws Exception
+	 * @throws GuzzleException
+	 */
+	public function adminCreatesGroupUsingTheGraphApi(string $group, ?string $user = null): void {
```

Comment:
```suggestion
	public function userCreatesGroupUsingTheGraphApi(string $group, ?string $user = null): void {
```
now this function can be used to create a group with any user, not only admin